src/main/java/com/ays/assignment/model/enums/AssignmentStatus.java

```diff
@@ -9,6 +9,7 @@ public enum AssignmentStatus {
     RESERVED,
     ASSIGNED,
     IN_PROGRESS,
-    DONE
+    DONE,
+    DELETED
```

Comment:
`DELETED` adında bir statüye şu an için ihtiyacımız yok, bu yüzden kaldırmamız daha iyi olabilir

Reply:
`DELETED` kısımı kaldırdım.